src/bsoncxx/CMakeLists.txt

```diff
@@ -65,6 +65,8 @@ add_library(bsoncxx SHARED
     ${bsoncxx_sources}
 )
 
+set(bsoncxx_libs ${LIBBSON_LIBRARIES})
+
```

Comment:
intentional, because I want to link the bsoncxx examples against the libs that bsoncxx links against. I figured it was good to put them in a var in case we ever add more.